test/coverage/bypass-stack-alt.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
I was going to have `bypass-stack.js` also have this functionality and let `getIsSet` be parameterized to the two `utilModule` copies, but I didn't immediately see how to reset the coverage state of the module.